pkg/plugins/trivy/plugin.go

```diff
@@ -319,7 +319,7 @@ func (p *plugin) Init(ctx trivyoperator.PluginContext) error {
 	})
 }
 
-func (p *plugin) GetScanJobSpec(ctx trivyoperator.PluginContext, workload client.Object, credentials map[string]docker.Auth) (corev1.PodSpec, []*corev1.Secret, error) {
+func (p *plugin) GetScanJobSpec(ctx trivyoperator.PluginContext, workload client.Object, credentials map[string]docker.Auth, securityContext *corev1.SecurityContext) (corev1.PodSpec, []*corev1.Secret, error) {
```

Comment:
Passing a *fifth* argument in this function isn't very nice - is there a better way?

Reply:
We could encapsulate it with a designated struct however it will required a bigger refactor , which maybe better to do it later